packages/lab/src/slider/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./Slider";
-export * from "./types";
+export * from "./RangeSlider";
+export * from "./internal/useSliderThumb";
```

Comment:
We should be careful that this isn't exposing this from the package unnecessarily, import {} from "@salt-ds/core"

Reply:
if it's in internal would say that it should not be shared ?
if it needs to be shared move it out of internal?